.github/workflows/build_apps.yml

```
@@ -0,0 +1,97 @@
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  DOCKER_IMAGE_REGISTRY: "ghcr.io"
+  DOCKER_USERNAME: "elie222"
+
+jobs:
+
+  set-version:
+    if: ${{ github.repository == format('{0}/inbox-zero', env.DOCKER_USERNAME) }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)